0001-typedb/02-docs.md

```diff
@@ -0,0 +1,63 @@
+---
+pageTitle: Documentation overview
+keywords: typedb, typeql, documentation, overview, introduction
+longTailKeywords: documentation overview, learn typedb, learn typeql, typedb schema, typedb data model
+summary: A birds-eye view of TypeQL and TypeDB
+toc: false
+---
+
+# Documentation overview
+
+This is the TypeDB server documentation section. It has the following sections:
+
+- **Getting started**
+  - [Introduction](01-start/01-introduction.md) — a brief description of TypeDB capabilities.
+  - [Installation](01-start/02-installation.md) — TypeDB server installation manual.
+  - [Quickstart guide](01-start/03-quickstart.md) — a guide to quickly set up a TypeDB database.
+  - [IAM schema explanation](01-start/04-iam-schema.md) — a brief description of IAM schema that widely used throughout 
+    the documentation.
+  - [Sample application](01-start/05-sample-app.md) — a simple example of how to use TypeDB with Java, Python, and 
+    Node.js.
+- **Development**
+  - [Connect](02-dev/01-connect.md) — TypeDB server and database connections
+  - [Schema](02-dev/02-schema.md) — types and rules definition
+  - [Matching patterns](02-dev/03-match.md) — patterns design and matching data
+  - [Writing data](02-dev/04-write.md) — data modification queries
+  - [Reading data](02-dev/05-read.md) — data retrieval queries
+  - [Inferring data](02-dev/06-infer.md) — data inference queries
+  - [Response interpretation](02-dev/07-response.md) — parsing query response
+  - [API and Drivers](02-dev/08-api.md) — brief description of Client API and TypeDB Drivers
+  - [Best practise](02-dev/09-best.md) — schema and query design tips
```

Comment:
"Best practices" or "Best Practices"—capitalisation is inconsistent here.

Reply:
Fixed inconsistencies in https://github.com/vaticle/docs/pull/676/commits/a6a9f9fd9aaab3423b1037b2c58e9575131e2ec0 (also in this PR is not a full version due to secondary files merge conflict. If you want, you can see full version at - https://github.com/izmalk/vaticle-docs/commit/f5e6a88d5a8a355a988856fb13e2315eae5226a9).

Be advised, that this particular example was not altered as it is correct that way. Best practices should be only first character capitalized. As it is here already. "API and Drivers" should be written that way due to capitalization of TypeDB Drivers (as a proper name).